tests/data/challenge_response.py

```diff
@@ -17,7 +17,7 @@
                             "description": "Excepturi eligendi minus modi delectus doloreasperiores voluptatem. \
                             Aspernatur itaque vitae repellendus. Natus ut tenetur labore dolores ex repudiandae.",
                             "enable_forum": true,
-                            "end_date": "2019-06-18T20:00:00Z",
+                            "end_date": "2100-06-18T20:00:00Z",
```

Comment:
I think we should use `timezone.now()` here. Is there any issue in using it?

Reply:
Thanks for the review :)
Resolved used `datetime.datetime.now()` as `timezone.now()` needs django.